tools/kmodules/sched_tp/Makefile

```diff
@@ -1,3 +1,31 @@
 obj-m += sched_tp.o
 
 EXTRA_CFLAGS = -I$(src)
+
+VMLINUX_DEPS_H = vmlinux_deps.h
+VMLINUX_H = vmlinux.h
+
+VMLINUX_DEPS_TXT = vmlinux_deps.txt
+VMLINUX_TXT = vmlinux.txt
+
+ifeq ($(wildcard $(KERNEL_SRC)/vmlinux), )
+	VMLINUX = /sys/kernel/btf/vmlinux
+else
+	VMLINUX = $(KERNEL_SRC)/vmlinux
+endif
+
+all: $(VMLINUX_H)
+	make -C $(KERNEL_SRC) M=$(PWD) modules
+
+install:
+	make -C $(KERNEL_SRC) M=$(PWD) modules_install
+
+clean:
+	make -C $(KERNEL_SRC) M=$(PWD) clean
+	rm -f $(VMLINUX_H) $(VMLINUX_DEPS_H)
+
+$(VMLINUX_DEPS_H): $(VMLINUX_DEPS_TXT) $(VMLINUX)
+	pahole -C file://vmlinux_deps.txt $(VMLINUX) > $@
```

Comment:
Looks like `file://vmlinux_deps.txt` should be replaced with `$(VMLINUX_DEPS_TXT)`

Reply:
Modified the code.